### ISSUE TITLE
Skupper console throwing CORS error.

### ISSUE THREAD

**Issue by rafaelvzago** [skupperproject/skupper#1133]
## Description

- Skupper is enabled with the console and flow collector in site 1 (namespace: public) and also enabled without the flow-collector and console at site 2 (namespace: private) and when I go to addresses I'm facing CORS errors at the developer console and also getting a CORS error screen.

## Steps to Reproduce

1. Install Skupper with `skupper init --ingress route --enable-console --enable-flow-collector` in the first namespace. (public)
2. Install Skupper with `skupper init ` in the first namespace (private).
3. Link both sites private -> public
4. Access the console and try to see the addresses

## Expected Behavior

Even the console running on the same cluster, the namespaces are different and I think we should be able to see it.

## Screenshots

![Screen Shot 2023-06-01 at 16 43 28](https://github.com/skupperproject/skupper/assets/1843938/8b6b86b8-0f2f-46d9-81cd-a542c667f81f)

## Environment (if relevant)

- Testing example: https://github.com/skupperproject/skupper-example-patient-portal
- Skupper Version site 1 (public): 
```
╰─ skupper version
client version                 1.4.0-rc2
transport version              quay.io/skupper/skupper-router:2.4.0-rc2 (sha256:9f60ac5f31cf)
controller version             quay.io/skupper/service-controller:1.4.0-rc2 (sha256:ee17c8d8eba8)
config-sync version            quay.io/skupper/config-sync:1.4.0-rc2 (sha256:64a2f3235092)
flow-collector version         quay.io/skupper/flow-collector:1.4.0-rc2 (sha256:5d966029f6c7)
```
- Skupper Version site 2 (private): 
```
╰─ skupper version
client version                 1.4.0-rc2
transport version              quay.io/skupper/skupper-router:2.4.0-rc2 (sha256:9f60ac5f31cf)
controller version             quay.io/skupper/service-controller:1.4.0-rc2 (sha256:ee17c8d8eba8)
config-sync version            quay.io/skupper/config-sync:1.4.0-rc2 (sha256:64a2f3235092)
flow-collector version         not-found
```
## Additional Information

Also, there is the HAR file of the test and also the console output log.
- [console-export-2023-6-1_16-32-56.txt](https://github.com/skupperproject/skupper/files/11629083/console-export-2023-6-1_16-32-56.txt)
- [skupper-public.apps.ocp412.rhaiqe.devcluster.openshift.com_Archive [23-06-01 16-31-59].zip](https://github.com/skupperproject/skupper/files/11629087/skupper-public.apps.ocp412.rhaiqe.devcluster.openshift.com_Archive.23-06-01.16-31-59.zip)

**Comment by ajssmith** [comment]
Presently, the console runs against the collector in a single namespace. 

What url did you access to try to run the console from another namespace?

**Comment by rafaelvzago** [comment]
Hello @ajssmith!

I'm accessing the URL provided by the skupper route created at the initialization. The issue here was bypassed when I manually accessed the Prometheus address I was getting the CORS exception:
- `https://skupper-prometheus-mydomain/api/v1/query?query=sum+by(address)(flows_total)`

After accepting the self-signed certificate from the Prometheus URL the console works as expected. I'm unsure if this is a desired behavior or if we must address this before the release. 

![Screen Shot 2023-06-02 at 13 35 35](https://github.com/skupperproject/skupper/assets/1843938/9f1f68fd-9875-4d4d-9b42-6745785c3c15)

**Comment by rafaelvzago** [comment]
Screenshot of the console error:

![image](https://github.com/skupperproject/skupper/assets/1843938/5902cc4a-c690-45bd-8ad0-5d06a3a4046f)
![Screen Shot 2023-06-05 at 16 00 11](https://github.com/skupperproject/skupper/assets/1843938/b4f6c4a6-95f0-4856-ab13-5ab8e2584115)

**Comment by kronotheone** [comment]
I seem to encounter a similar issue. 
Using 1.4.0-rc2, Ive created a skupper site:
./skupper init --console-auth openshift --enable-console --enable-flow-collector --ingress loadbalancer --router-load-balancer-ip 10.xxx.137 --controller-load-balancer-ip 10.xxx.138 --console-ingress route

The skupper site is created and I have console access.
However, when going to the addresses tab, I get a CORS warning, see screenshot:

![image](https://github.com/skupperproject/skupper/assets/30655630/9add845d-03ed-484b-8795-cd8b66bf5c0a)

**Comment by ajssmith** [comment]
addressed by #1140 as prom query moved behind api and server no longer exposed
